FlowCryptUITests/SignInImapTest.swift

```diff
@@ -43,6 +40,102 @@ extension SignInImapTest {
         wait(4)
         XCTAssert(app.buttons["+"].exists)
     }
+    
+    // restart app -> loads inbox
+    func test_2_restart_app_load_inbox() {
+        let application = XCUIApplication()
+        wait(2)
+        application.buttons["+"].tap()
+        
+        wait(0.3)
+        
+        application.typeText("ElonMusk@gmail.com")
+        application.tables.textFields["Subject"].tap()
+        wait(3)
+        application.tables.textFields["Subject"].tap()
+        
+        application.typeText("SpaceX")
```

Comment:
Our source code is public, this should be changed to something neutral.

Reply:
Also, whenever possible, test emails should either used `.test` TLD, or `example.com` domain, so that we don't accidentally send email to an actual email server during tests.